tests/storage.test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var request = require('supertest');
+var should = require('should');
+var assert = require('assert');
+var load = require('./fixtures/load');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['load' is defined but never used.](https://www.codacy.com/app/jasoncalabrese/cgm-remote-monitor/file/759804173/issues/source?bid=2449627&fileBranchId=2449709#l6)